### PR TITLE
Feat : Add to playlist

### PR DESCRIPTION
**Pull request by shlok30**
Hi Team,

I have added playlist functionality to my app.(Auth hasn't been added yet)

Demo Link : https://water-videos-git-add-to-playlist-shlok30.vercel.app/

Dummy Login Credentials 
Email : test@gmail.com
password : test123

Video Demo:

https://user-images.githubusercontent.com/60795588/169474042-cd8a6e66-bda1-481c-a757-4652ff61c847.mov

Can you please review my code?

Thanks

### CODE REVIEW

**Review by sashaboi — APPROVED** [review]
Code is very readable and review friendly, 
Liked the folder structure , it clearly shows how the data is flowing.
Looks good to merge!

File: src/App.js [hunk 1]
```diff
@@ -23,6 +25,9 @@ function App() {
         <Route path = "/signup" element = {<Signup />} />
         <Route path = "/watch-later" element = {<WatchLaterPage />} />
         <Route path = "/liked-videos" element = {<LikedVideos />} />
+        <Route path = "/playlists" element ={<PlaylistListing />} />
+        <Route path = "/playlists/:videoId" element ={<PlaylistListing />} />
+        <Route path = "/playlist/:playlistId" element = {<PlaylistPage />} />
```
Comment (SHIKHASINGH1506) [inline src/App.js:30]: you can nest playlists/:videoId route inside playlists route.
You can read more about it here  [https://reactrouter.com/docs/en/v6/getting-started/overview#nested-routes](url)

File: src/components/video-card.js [hunk 2]
```diff
@@ -1,13 +1,17 @@
...
 import { useUser } from "../context/user-context"
 
 const VideoCard = ({width,id,title,creator,views}) => {
 
     const location = useLocation()
 
-    console.log("location path",location.pathname)
+    const {playlistId} = useParams()
+
+    //console.log("location path",location.pathname)
```
Comment (SHIKHASINGH1506) [inline src/components/video-card.js:10]: remove commented code

**Review by SHIKHASINGH1506 — APPROVED** [review]
1. code looks modular
2. added one suggestion 
3. can be merged

**Pull request merged**